GlassBR - probBr inserted twice in SymbolMap

The UID `probBr` is inserted twice into the GlassBR `SymbolMap`.

The code that adds terms to the `symbMap` `ChunkDB` that is causing the issue is here:
https://github.com/JacquesCarette/Drasil/blob/0ab5dec39cad09265bc57ad36805d980f6337296/code/drasil-example/glassbr/lib/Drasil/GlassBR/Body.hs#L134-L143

`thisSymbols` is the function that creates the SymbolMap where the problem is occuring, and I believe I have isolated where `probBr` is being added twice.

`thisSymbols` adds the UIDs from the `iMods` list and the `symbolsForTable` list, both of which contain `probBr`

**Location 1:**

`thisSymbols` function:
https://github.com/JacquesCarette/Drasil/blob/0ab5dec39cad09265bc57ad36805d980f6337296/code/drasil-example/glassbr/lib/Drasil/GlassBR/Symbols.hs#L21-L25

which adds `iMods`:
https://github.com/JacquesCarette/Drasil/blob/0ab5dec39cad09265bc57ad36805d980f6337296/code/drasil-example/glassbr/lib/Drasil/GlassBR/IMods.hs#L22-L24

`iMods` contains `probOfBreak`, which then has `probBr` as an output (if I am interpreting it correctly)
https://github.com/JacquesCarette/Drasil/blob/0ab5dec39cad09265bc57ad36805d980f6337296/code/drasil-example/glassbr/lib/Drasil/GlassBR/IMods.hs#L136-L139


**Location 2:**

`symbolsForTable`, added in `thisSymbols`, has a list `outputs` added to it, which contains `probBr` as one of the terms as well
https://github.com/JacquesCarette/Drasil/blob/0ab5dec39cad09265bc57ad36805d980f6337296/code/drasil-example/glassbr/lib/Drasil/GlassBR/Symbols.hs#L16-L19

`outputs` then contains `probBr`:
https://github.com/JacquesCarette/Drasil/blob/0ab5dec39cad09265bc57ad36805d980f6337296/code/drasil-example/glassbr/lib/Drasil/GlassBR/Unitals.hs#L115-L116


Like we discussed in the meeting, the issue is determining which probBr should be the correct one to remove.

Removing `map (^. output) iMods` and `  ++ (map asVC (concatMap (\(Mod _ _ _ _ l) -> l) allMods) \\ symbolsForTable)` seems to also remove something that is needed, I get this error:

```console
glassbr.EXE: Symbol: read_table not found in SymbolMap
CallStack (from HasCallStack):
  error, called at lib\Database\Drasil\ChunkDB.hs:116:24 in drasil-database-0.1.1.0-4jdMY7TWows9S8CmX9loPI:Database.Drasil.ChunkDB
make: *** [Makefile:311: glassbr_gen] Error 1
```

For reference this is what the `thisSymbol` function looks like (in case you meant something else)
```haskell
thisSymbols :: [QuantityDict]
thisSymbols = map qw implVars ++ symbolsForTable
```

--- a/code/drasil-example/glassbr/lib/Drasil/GlassBR/Symbols.hs
+++ b/code/drasil-example/glassbr/lib/Drasil/GlassBR/Symbols.hs
@@ -18,8 +18,7 @@ symbolsForTable = inputs ++ outputs ++ tmSymbols ++ map qw specParamVals ++
   map qw symbolsWithDefns ++ map qw symbols ++ map qw unitless ++
   map qw inputDataConstraints ++ interps
 
-thisSymbols :: [QuantityDict]
-thisSymbols = map (^. output) iMods
   -- include all module functions as symbols
-  ++ (map asVC (concatMap (\(Mod _ _ _ _ l) -> l) allMods) \\ symbolsForTable)
-  ++ map qw implVars ++ symbolsForTable
+thisSymbols :: [QuantityDict]
+thisSymbols = (map asVC (concatMap (\(Mod _ _ _ _ l) -> l) allMods)
+  \\ symbolsForTable) ++ map qw implVars ++ symbolsForTable

